.buildkite/scripts/dra_release.sh

```diff
@@ -33,15 +32,21 @@ with_go
 
 with_mage
 
-if [[ "${TYPE}" == "snapshot" ]]; then
-    SNAPSHOT=true
-fi
-
-mkdir -p ${BASE_DIR}/reports
-./dev-tools/dependencies-report --csv ${BASE_DIR}/reports/dependencies-${VERSION}.csv
-cd ${BASE_DIR}/reports && shasum -a 512 dependencies-${VERSION}.csv > dependencies-${VERSION}.csv.sha512
+case "${TYPE}" in
+    "snapshot")
+        make release-manager-dependencies-snapshot
+        make release-manager-snapshot
+        ;;
+    "staging")
+        make release-manager-dependencies-release
+        make release-manager-release
+        ;;
```

Comment:
`make release-manager-snaphot` and `make release-manager-release` should be part of the `.buildkite/scripts/package.sh` in the step "Package Ubuntu-20 x86_64"